docs/usage.rst

```diff
@@ -50,6 +50,12 @@ that in some specific cases `XPath`_ is more powerful than `CSS`_.
 .. _XPath: http://www.w3.org/TR/xpath
 .. _CSS: http://www.w3.org/TR/selectors
 
+.. note::
+    HTML comments mentioned within <style> and <script> tags will be treated as raw text.
+    The reason is that it is a common practice to comment out the content of <style> 
```

Comment:
This is more like it, yes! :+1: 
Except I said it wrong, I said HTML/XML but i mean only HTML: it's not a concern of the **HTML** parser (which knows that a `script` is application-specific).
The **XML** parser doesn't know what `script` is and handles like any other tag, so it will actually parse the comments, as demonstrated by [the workaround for parsing these cases shown in this comment](https://github.com/scrapy/parsel/issues/43#issuecomment-229134972). =)

So, this note should mention that workaround (essentially, parsing the script content using `type="xml"`) for completeness.